packages/http-request-handler/src/createSimpleServer.helper.ts

```diff
@@ -0,0 +1,63 @@
+import http from 'http';
```

Comment:
I believe it belongs to spec/testing part of `@coolio/http` - import it from there (can be a direct `@coolio/http/dist/createSimpleServer.helper` import)